tools/EMLassemblyline/eml2eal.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="eml2eal" name="EML to EAL" version="0.1.0+galaxy0" profile="22.05">
+    <description> Create EAL templates from EML metadata </description>
+    <requirements>
+       <requirement type="package" version="3.5.5">r-emlassemblyline</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+       Rscript
+         '$__tool_directory__/eml2eal.R'
+         '$inputmetadatafile'
+    ]]></command>
+    <inputs>
+        <param name="inputmetadatafile" type="data" format="xml" label="EML metadata file" help="Insert only metadata using EML standard"/>
+    </inputs>
+    <outputs>
+        <collection name="EML_templates" type="list" label="EML Templates">

```

Comment:
How many outputs are generated? If only one XML file is ever generated, this is overkill and not needed. Simply add a `<data >` tag.